src/client/http_client.rs

```diff
@@ -336,6 +338,61 @@ impl HTTPClient {
         })
     }
 
+    pub async fn analytics_query_stream_request(
```

Comment:
Can we just use analytics_query_request and always return a streaming response? It should be fairly to add a helper to consume the stream at a higher level for things like index management.